app/static/js/vendor/jquery/jquery.checkboxes-1.2.0.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
```

Comment:
Yes. Exclude it. We already have too many libraries for even simple things ... Disabled cursor over hovering can even be done with simple jQuery/css